### PR TITLE
Feat(Mangas-Origines): add website support

### PR DESCRIPTION
**Pull request by Oneloutre**
Adding Mangas-Origines to supported websites

Signed-off-by: Onelots <54822802+Oneloutre@users.noreply.github.com>

## Description 
Adding support for mangas-origines website

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
@Oneloutre struggled with getting the images to load so here:

**Comment by Oneloutre** [comment]
thanks @DarkVillager, I dunno why it wasn't showing up 👍

**Comment by EncryptedDev** [comment]
I'm baffled, was your [comment](https://github.com/PreMiD/Presences/pull/7110#issuecomment-1374428702) suggesting something hasn't been fixed or a thanks for fixing it? @DarkVillager

**Comment by Oneloutre** [comment]
> I'm baffled, was your [comment](https://github.com/PreMiD/Presences/pull/7110#issuecomment-1374428702) suggesting something hasn't been fixed or a thanks for fixing it? @DarkVillager

I dunno why, I couldn't upload images 🤷 so he did it for me

File: websites/M/Mangas-Origines/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"service": "Mangas-Origines",
+	"description": {
+		"en": "Read Comics on Mangas-Origines."
+	},
+	"url": [
+		"mangas-origines.fr",
+		"x.mangas-origines.fr"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/MqvvhSw.png",
```
Comment (Bas950) [inline websites/M/Mangas-Origines/metadata.json:16]: Logo must be 512x512

File: websites/M/Mangas-Origines/presence.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
+const presence = new Presence({
+		clientId: "1059611780520874044",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const { pathname, href } = window.location,
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/MqvvhSw.png",
```
Comment (Bas950) [inline websites/M/Mangas-Origines/presence.ts:10]: `largeImageKey` must be 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Mangas-Origines/metadata.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
 	],
 	"version": "1.0.0",
 	"logo": "https://i.imgur.com/MqvvhSw.png",
-	"thumbnail": "https://i.imgur.com/NInXya0.png",
+	"thumbnail": "https://i.imgur.com/KZ3FSWl.png",
```
Comment (DarkVillager) [inline websites/M/Mangas-Origines/metadata.json:17]: you changed the thumbnail not the logo

**Comment by EncryptedDev** [comment]
The presence seems to only support a handful of pages, hence why I've closed the PR due to it being low quality

**Pull request closed without merging**